src/ert/gui/model/snapshot.py

```diff
@@ -356,10 +373,12 @@ def data(self, index: QModelIndex, role: int = Qt.ItemDataRole.DisplayRole) -> A
         }:
             return QColor()
 
-        return QVariant()
+        return None
 
     @staticmethod
-    def _real_data(_: QModelIndex, node: RealNode, role: int) -> Any:
+    def _real_data(
+        _: QModelIndex | QPersistentModelIndex, node: RealNode, role: int
+    ) -> Any:
```

Comment:
Another Any

Reply:
it is only used by the data() method which returns any